src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -277,4 +291,16 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
 
         return discounts;
     }
+
+    private _getCategories(categories?: LineItemCategory[][]): string[][] {
+        if (!categories) {
+            return[[]];
+        }
+
+        return categories.map(categoryTree => {
+            return categoryTree.map( category => {
```

Comment:
There is an extra space before the category variable, you can also use the arrow function short syntax

```
categories.map(categoryTree => categoryTree.map(category => category.name))
```

Reply:
👍🏼